common/persistence/class.InMemoryAdvKvDriver.php

```diff
@@ -40,11 +48,9 @@ public function hExists($key, $field)
 
     public function hSet($key, $field, $value)
     {
-        if (! isset($this->persistence[$key])) {
-            return false;
-        }
+        $result = !isset($this->persistence[$key][self::HPREFIX . $field]);
```

Comment:
if you do not check that array key exists it could be a php warning with ba consequences

Reply:
No, using `isset()` it's allowed to check existence of key in multidimensional array.